app/code/community/SomethingDigital/Modal/Block/Modal.php

```diff
@@ -12,6 +12,9 @@ public function getConfig($value)
     public function getModalContent() 
     {
         $blockId = $this->getConfig('block');
+        if(!$this->getLayout()) {
+            return;
```

Comment:
I don't think it's necessary at all. Not my other comment. They weren't on the latest version.